### PR TITLE
Added in DNS and SQL cmdlets, and parameter set tables.

### PR DESCRIPTION
**Pull request by marceloyuela**
Medium size PR here. Although the scope of changes is encapsulated by the title of the PR.

### CODE REVIEW

File: website_v2/gcloud-powershell-website-v2/www/static/app/app.js [hunk 1]
```diff
@@ -32,6 +32,7 @@
     });
 
     /* TODO: Change templates from html to .ng */
+    /** Template for the table of contents. **/
```
Comment (iantalarico) [inline website_v2/gcloud-powershell-website-v2/www/static/app/app.js]: Nit: this is a directive, the template is the ng file

**Comment by iantalarico** [comment]
JS and HTML LGTM.  Someone else should look at the PS bits though.

**Comment by marceloyuela** [comment]
@chrsmith @ILMTitan Can I get a look at the PS bits?

**Comment by ILMTitan** [comment]
PS bits LGTM.

**Pull request merged**